src/vitest.js

```diff
@@ -0,0 +1,30 @@
+// @ts-check
+
+import vitestPlugin from '@vitest/eslint-plugin';
+import {ERROR, testFiles} from './config.js';
+
+/**
+ * @type {Array<import('eslint').Linter.Config>}
+ */
```

Comment:
same as above regarding `it.only`

Reply:
See discussion [above](https://github.com/molindo/eslint-config-molindo/pull/109/files#r1800703840).